akd/src/append_only_zks.rs

```diff
@@ -413,39 +381,29 @@ impl Azks {
 
             unchanged.push(Node::<H> {
                 label: node.label,
-                hash: node
-                    .get_value_without_label_at_epoch::<_, H>(storage, node.get_latest_epoch())
-                    .await?,
+                hash: to_digest::<H>(&node.hash)?,
             });
             return Ok((unchanged, leaves));
         }
-        if node.get_birth_epoch() > end_epoch {
-            // really you shouldn't even be here. Later do error checking
-            return Ok((unchanged, leaves));
-        }
+        // if node.get_birth_epoch() > end_epoch {
```

Comment:
nit: From the comment it looks like this wasn't necessary anyways. Can we cleanup this artifact?